content/shmem_atomic_compare_swap.tex

```diff
@@ -5,14 +5,16 @@
 \begin{apidefinition}
 
 \begin{C11synopsis}
-TYPE shmem_cswap(TYPE *dest, TYPE cond, TYPE value, int pe);
+TYPE shmem_atomic_compare_swap(TYPE *dest, TYPE cond, TYPE value, int pe);
```

Comment:
The change from `cswap` to `compare_swap` was to be stylistically closer to the C11 [`atomic_compare_exchange`](http://en.cppreference.com/w/cpp/atomic/atomic_compare_exchange).